### DISCUSSION TITLE
Setting radii type in SPACEFILL

### DISCUSSION THREAD

**Discussion by jfredrikhaeffner** [TinkerTools/tinker#158] [General]
How can one set the radii type (BONDI) in the key file when conducting a SPACEFILL calculation?

**Comment by jayponder** [comment]
Having just checked the source code, it appears that the current SPACEFILL program will always use atomic radii based on the vdw radii set by the current force field. There is no way, by keyword, to use the “Bondi” radii in place of the force field radii. The only way to do this without changing the source code would be to set the vdw radii for the force field being used to be equal to the Bondi values (using "vdw" keywords) prior to running SPACEFILL.

Also, Tinker does internally store a generic set of vdw radii by element. These generic vdw radii are actually a generalization of the original Bondi radii, but identical or similar to Bondi values for the elements in the original Bondi set. It would be a trivial change to the SPACEFILL program (source file spacefill.f) to make it use these internally stored values, or to select via a keyword.
